### PR TITLE
feat(admin-ui): wrap sidebar with scroll

### PR DESCRIPTION
**Pull request by lundeeell**

### CODE REVIEW

File: packages/engine-ui/src/components/Sidebar.tsx [hunk 1]
```diff
@@ -19,9 +20,21 @@ const Container = styled.div`
 const Content = styled.div`
   padding: 2rem;
   width: 350px;
+  overflow: auto;
 `
 
-const Sidebar = (state) => {
+interface props {
```
Comment (momentiris) [inline packages/engine-ui/src/components/Sidebar.tsx:26]: Should this be capitalized, for convention?

File: packages/engine-ui/src/types.ts [hunk 2]
```diff
@@ -23,3 +23,33 @@ export type Vehicle = {
 }
 
 export type NotificationType = Vehicle | Booking
+
+export interface PlanVehicle {
```
Comment (momentiris) [inline packages/engine-ui/src/types.ts:27]: What is PlanVehicle? Is there anything better to name this? :)

**Review by momentiris — APPROVED** [review]

File: packages/engine-ui/src/components/Sidebar.tsx (hunk 1 above)
Comment (lundeeell) [reply packages/engine-ui/src/components/Sidebar.tsx:26]: Yes, will fix!

File: packages/engine-ui/src/types.ts (hunk 2 above)
Comment (lundeeell) [reply packages/engine-ui/src/types.ts:27]: It is...vehicle in plan. I didn't create this in this PR, only moved it. But we should go over types and naming in this project

Comment (momentiris) [reply packages/engine-ui/src/types.ts:27]: Okay. Makes me think of something that is both a vehicle and a plan at the same time.. dogduck.

**Pull request merged**